libs/trackers/tracker.py

```diff
@@ -0,0 +1,32 @@
+class Track:
```

Comment:
What about calling this file `track.py`?

Reply:
`track.py` would be better.